packages/typespec-client-generator-core/test/types/general-decorators-list.test.ts

```diff
@@ -327,4 +327,29 @@ describe("typespec-client-generator-core: general decorators list", () => {
       ]);
     });
   });
+
+  describe("csharp only decorator", () => {
```

Comment:
add test cases to verify that if you don't pass in scope param it doesn't get applied to java but does get applied to csharp.  Also add test case that verifies the opposite that if you override scope to java it applies to java but not csharp.

Reply:
Will be covered in https://github.com/Azure/typespec-azure/issues/1295